lynx/common/actions/common_requirements.py

```diff
@@ -0,0 +1,19 @@
+from typing import Callable
+
+from lynx.common.object import Object
+from lynx.common.vector import Vector
+
+
+class CommonRequirements:
+
+    @staticmethod
+    def is_walkable_destination(object_id: int, vector: Vector):
```

Comment:
could we change it to more generic requirement
```python
is_walkable(position: Vector)
```
so that we can use it with any action that has to do with walkability